backend/app/views/spree/admin/orders/_carton_manifest.html.erb

```diff
@@ -4,7 +4,7 @@
       <%= render 'spree/admin/shared/image', image: item.variant.display_image, size: :mini %>
     </td>
     <td class="item-name">
-      <%= item.variant.product.name %><br><%= "(" + variant_options(item.variant) + ")" unless item.variant.option_values.empty? %>
+      <%= link_to item.variant.product.name, edit_admin_product_path(item.variant.product), target: "_blank" %><br><%= "(" + variant_options(item.variant) + ")" unless item.variant.option_values.empty? %>
```

Comment:
Looks great, but I'd rather not use `target: "_blank"` here. Links should open in the same tab.

Reply:
Ok, I think when someone works on an order he will want to keep it open while looking at a product details quickly but doesn't matter. I changed it.